### PR TITLE
Add weekly update 2016-07-08

### PR DESCRIPTION
**Pull request by JungMinu**
Please review this post. :heart:,@nodejs/evangelism

### CODE REVIEW

File: locale/en/blog/weekly-updates/weekly-update.2016-07-08.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+### Community Updates
+
+* [Node Hero - Debugging Node.js Applications](https://blog.risingstack.com/node-hero-node-js-debugging-tutorial/): you can learn how to get started with Node.js and deliver software products using it.
+* [Node Hero - Node.js Unit Testing Tutorial](https://blog.risingstack.com/node-hero-node-js-unit-testing-tutorial/): you can learn how to get started with Node.js and deliver software products using it.
+If you have spotted or written something about Node.js, do come over to our [Evangelism team repo](https://github.com/nodejs/evangelism) and suggest it on the [Issues page](https://github.com/nodejs/evangelism/issues), specifically the Weekly Updates issue.
+
+### Upcoming Events
+
+* [NodeConf Argentina](https://2016.nodeconf.com.ar): 18 - 19 November, 2016
```
Comment (MylesBorins) [inline locale/en/blog/weekly-updates/weekly-update.2016-07-08.md:35]: should we include [nodeconf.eu](http://www.nodeconf.eu/)?

Comment (ghost) [inline locale/en/blog/weekly-updates/weekly-update.2016-07-08.md:35]: it's earlier than nodeconf ar, so why not?

File: locale/en/blog/weekly-updates/weekly-update.2016-07-08.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+### Binaries for Linux on Power Systems
+
+Community downloads now include binaries for Linux on Power Systems with big endian in addition to the existing little endian binaries.
+
+See https://nodejs.org/en/download/.
+
+### Community Updates
+
+* [Node Hero - Debugging Node.js Applications](https://blog.risingstack.com/node-hero-node-js-debugging-tutorial/): you can learn how to get started with Node.js and deliver software products using it.
+* [Node Hero - Node.js Unit Testing Tutorial](https://blog.risingstack.com/node-hero-node-js-unit-testing-tutorial/): you can learn how to get started with Node.js and deliver software products using it.
```
Comment (evanlucas) [inline locale/en/blog/weekly-updates/weekly-update.2016-07-08.md]: the part after the `:` is the same as the previous link. Is that intentional?

Comment (JungMinu) [inline locale/en/blog/weekly-updates/weekly-update.2016-07-08.md]: @evanlucas it was a mistake, will fix it

Comment (JungMinu) [inline locale/en/blog/weekly-updates/weekly-update.2016-07-08.md]: @evanlucas fixed and updated :)

**Comment by JungMinu** [comment]
Added NodeConf EU 2016 and updated :)

File: locale/en/blog/weekly-updates/weekly-update.2016-07-08.md (hunk 2 above)
Comment (JungMinu) [inline locale/en/blog/weekly-updates/weekly-update.2016-07-08.md]: fixed and updated :)

**Comment by Knighton910** [comment]
LGTM

**Comment by lpinca** [comment]
Maybe it's only an OCD of mine but can we squash these commits the next time?
`Update weekly-update.2016-07-08.md` x5 😕.

**Comment by JungMinu** [comment]
Yup, I will squash these commits

**Pull request merged**